gengine/app/leaderboard.py

```diff
@@ -13,28 +14,29 @@ class GlobalLeaderBoardSubjectSet:
     @classmethod
     def forward(cls, subjecttype_id, from_date, to_date, whole_time_required):
         q = select([t_subjects.c.id, ]).where(t_subjects.c.subjecttype_id == subjecttype_id)
-        if whole_time_required:
-            q = q.where(and_(
-                t_subjects.c.created_at <= from_date
-                #or_(
-                #    t_subjects.c.deleted_at == None,
-                #    t_subjects.c.deleted_at >= to_date
-                #)
-            ))
-        else:
-            q = q.where(or_(
-                and_(
-                    t_subjects.c.created_at <= from_date,
+        if from_date != None and to_date != None:
```

Comment:
__[Flake8]__ comparison to None should be 'if cond is not None:' ([see on Sider](https://sider.review/gh/repos/31541379/pulls/46#issue-4893295))

<details>

| Rule |
| - |
| E711 |

</details>
<sub>posted by Sider</sub>